src/NearOrg/DataAvailabilityPage.jsx

```diff
@@ -238,6 +387,17 @@ return (
           </Text>
         </Flex>
 
+        {/* This will be uncommented once we get the updated image from the design team */}
```

Comment:
Are we waiting on another updated image? I'm wondering if this and the other commented code can be removed

Reply:
![Uploading Near Org Data Availability (4).png…]()

Reply:
@charleslavon image updated.

![Near Org Data Availability (4)](https://github.com/near/near-discovery-components/assets/1408936/e432c71f-e89a-4cde-844c-4691b0d2fa7c)